Dockerfile.rhel7

```diff
@@ -5,6 +5,7 @@ ENV GO_PACKAGE github.com/openshift/console-operator
 RUN go build -ldflags "-X $GO_PACKAGE/pkg/version.versionFromGit=$(git describe --long --tags --abbrev=7 --match 'v[0-9]*')" -tags="ocp" -o console ./cmd/console
 
 FROM registry.svc.ci.openshift.org/ocp/4.6:base
+RUN yum install -y python3 && yum clean all
```

Comment:
Is this needed? It's for the console operator image, not for the `downloads` deployment, which uses `registry.svc.ci.openshift.org/openshift:cli-artifacts`

Reply:
oh, I see what you're saying.  yeah, good point, we'll need to add that to cli-artifacts instead.